java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java

```diff
@@ -3008,10 +3027,194 @@ private void insideBreakOrContinue(Env env) throws IOException {
             }
         }
     }
+    
+    private void addClassTypes(final Env env, DeclaredType baseType) throws IOException{
```

Comment:
The baseType parameter is unused. Suggest to either pass it to addTypes, or drop it.

Reply:
thanks for correcting me, I will pass the parameter to to addTypes call